examples/visualisations/utils.py

```diff
@@ -194,28 +217,26 @@ def propose_market(
         str:
             Market id.
     """
-    vega.update_network_parameter(
-        wallet_name=AUX_PARTY_A.wallet_name,
-        proposal_key=AUX_PARTY_A.key_name,
-        parameter="market.fee.factors.infrastructureFee",
-        new_value="0.0",
-    )
-    vega.wait_for_total_catchup()
-    vega.update_network_parameter(
-        wallet_name=AUX_PARTY_A.wallet_name,
-        proposal_key=AUX_PARTY_A.key_name,
-        parameter="market.fee.factors.makerFee",
-        new_value="0.0",
+    market_config = MarketConfig()
+    market_config.set("instrument.name", "XYZ:DAI Visualisation Example")
+    market_config.set("instrument.code", "XYZ:DAI")
+    market_config.set("instrument.future.settlement_asset", asset_id)
+    market_config.set("instrument.future.quote_name", "fDAI")
+    market_config.set("instrument.future.number_decimal_places", 4)
+    market_config.set(
+        "instrument.future.terminating_key",
+        vega.wallet.public_key(
+            wallet_name=AUX_PARTY_A.wallet_name,
+            name=AUX_PARTY_A.key_name,
+        ),
     )
-    vega.wait_for_total_catchup()
-    vega.create_simple_market(
-        market_name="XYZ:DAI Visualisation Example",
-        wallet_name=AUX_PARTY_A.wallet_name,
-        proposal_key=AUX_PARTY_A.key_name,
-        termination_wallet_name=AUX_PARTY_A.wallet_name,
-        termination_key=AUX_PARTY_A.key_name,
-        settlement_asset_id=asset_id,
-        market_decimals=3,
+    if not price_monitoring:
+        market_config.set("price_monitoring_parameters.triggers", [])
```

Comment:
Does this work ok? I remember in the past a market with no price monitoring parameters at all did not really behave well.

Reply:
Seems to work okay but have realised my reasoning for doing it is flawed anyway...

Thought `create_simple_market` which this was replacing didn't use price monitoring.. will update it to set the current defaults.